tensorflow/contrib/distributions/python/kernel_tests/inverse_gamma_test.py

```diff
@@ -295,16 +296,18 @@ def testInverseGammaNonPositiveInitializationParamsRaises(self):
     with self.cached_session():
       alpha_v = constant_op.constant(0.0, name="alpha")
       beta_v = constant_op.constant(1.0, name="beta")
-      inv_gamma = inverse_gamma.InverseGamma(
-          concentration=alpha_v, rate=beta_v, validate_args=True)
-      with self.assertRaisesOpError("alpha"):
-        inv_gamma.mean().eval()
+      with self.assertRaisesWithPredicateMatch(errors.InvalidArgumentError,
+                                               "alpha"):
+        inv_gamma = inverse_gamma.InverseGamma(
```

Comment:
Unused variable 'inv_gamma' [unused-variable]

Reply:
Changed `inv_gamma` to `_`.